aerospike/datadog_checks/aerospike/data/conf.yaml.example

```diff
@@ -1,33 +1,55 @@
 instances:
+  
+  ## @param host - string - required
+  ## Aerospike host to connect to.
+  #
   - host: localhost
+  
+  ## @param port - integer - required
+  ## Aerospike port to connect to.
+  #
     port: 3003
 
-    # All metrics collected by default. You probably don't want to do this,
-    # so list the ones you do want here. Check with asinfo -v statistics -l
-    # and asinfo -v namespace/YOURNAMESPACE -l.
-
-    # metrics:
-    # - migrate_rx_objs
-    # - migrate_tx_objs
-    # namespace_metrics:
-    # - free-pct-disk
-    # - free-pct-memory
-    # - evicted-objects
-    # - expired-objects
-    # - master-objects
-    # - migrate-rx-partitions-initial
-    # - migrate-rx-partitions-remaining
-    # - migrate-tx-partitions-initial
-    # - migrate-tx-partitions-remaining
-
-    # tags:
-    # - optional_tag1
-    # - optional_tag2
+  ## @param tags  - list of key:value elements - optional
+  ## List of tags to attach to every metric, event and service check emitted by this integration.
+  ##
+  ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+  #
+  #  tags:
+  #    - <KEY_1>:<VALUE_1>
+  #    - <KEY_2>:<VALUE_2>
+    
+  ## @param metrics - list of strings - optional
+  ## All metrics collected by default. You probably don't want to do this,
```

Comment:
This was the description in the former conf file.. But i'm not sure of what it means.. `All metrics collected by default. You probably don't want to do this,` ? What are we expecting from the user here?

Reply:
🤷‍♂️